### PR TITLE
Spinner color fix.

### LINKED ISSUE #283: Toolbar Spinner is black on < Lollipop
Here is the screenshot from Kitkat device

screen shot 2015-03-03 at 23 57 01

### PR DESCRIPTION
**Pull request by tasomaniac**
Finally fixes #283 

Here are the screenshots from Kitkat and Lollipop devices. 

- Removed bunch of unused resource. 
- Removed custom implementation with `whiteText` variable. It is now done with only styles. 

screen shot 2015-07-25 at 13 18 09
screen shot 2015-07-25 at 13 17 14

### CODE REVIEW

**Comment by PareshMayani** [comment]
Said is on fire :) :+1:

**Comment by tasomaniac** [comment]
@PareshMayani thank you :smile:

**Comment by tasomaniac** [comment]
Build is fixed now.

File: app/src/main/java/org/gdg/frisbee/android/chapter/ChapterAdapter.java [hunk 1]
```diff
@@ -32,33 +29,12 @@
...
-            return view;
-        }
+    public ChapterAdapter(Context context, @LayoutRes int layoutRes) {
+        super(context, layoutRes);
     }
 
-
     @Override
-    public void addAll(Collection<? extends Chapter> chapters) {
+    public void addAll(@NonNull Collection<? extends Chapter> chapters) {
```
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/chapter/ChapterAdapter.java]: Better only annotate @Nullable and assume everything else as non null. Or is it different here?

Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/chapter/ChapterAdapter.java]: The parent class required @NonNull Lint warned me to use it.

**Pull request merged**